packages/compass-connections-navigation/src/sc-connections-navigation-tree.spec.tsx

```diff
@@ -1,458 +0,0 @@
-/* eslint-disable @typescript-eslint/no-empty-function */
-import React from 'react';
-import {
-  render,
-  screen,
-  cleanup,
-  within,
-  waitFor,
-} from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import { expect } from 'chai';
-import Sinon from 'sinon';
-import { ConnectionsNavigationTree } from './connections-navigation-tree';
-import type { PreferencesAccess } from 'compass-preferences-model';
-import { createSandboxFromDefaultPreferences } from 'compass-preferences-model';
-import { PreferencesProvider } from 'compass-preferences-model/provider';
-
-const databases = [
-  {
-    _id: 'foo',
-    name: 'foo',
-    collectionsStatus: 'initial',
-    collectionsLength: 5,
-    collections: [],
-  },
-  {
-    _id: 'bar',
-    name: 'bar',
-    collectionsStatus: 'ready',
-    collectionsLength: 3,
-    collections: [
-      { _id: 'bar.meow', name: 'meow', type: 'collection' },
-      { _id: 'bar.woof', name: 'woof', type: 'timeseries' },
-      { _id: 'bar.bwok', name: 'bwok', type: 'view' },
-    ],
-  },
-];
```

Comment:
I'm not sure if we want to remove this, because we want to be careful not to break the legacy functionality + there are still differences between the two versions. Replacing the databases and expanded props should fix the tests.

Reply:
You mean the file?

Reply:
Yes, I left this test suite as regression tests for the SC version.

Reply:
Actually it will need also the preferences and the connectionIds in the actions, so perhaps not that easy to fix. I don't know if it's worth it, I'm just feeling cautious around the SC version, since this is the part that's not behind the feature toggle but also we give less attention to it